app/src/main/java/org/mifos/mobile/theme/Theme.kt

```diff
@@ -0,0 +1,46 @@
+package org.mifos.mobile.theme
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+
+private val LightThemeColors = lightColors(
+    primary = Blue600,
+    primaryVariant = Blue400,
+    onPrimary = Black2,
+    secondary = Color.White,
+    secondaryVariant = Teal300,
+    onSecondary = Color.Black,
+    error = RedErrorDark,
+    onError = RedErrorLight,
+    background = BackgroundLight,
+    onBackground = Color.Black,
+    surface = Color.White,
+    onSurface = Black2,
+)
+
+private val DarkThemeColors = darkColors(
+    primary = Blue700,
+    primaryVariant = Color.White,
+    onPrimary = Color.White,
+    secondary = Black1,
+    onSecondary = Color.White,
+    error = RedErrorLight,
+    background = BackgroundDark,
+    onBackground = Color.White,
+    surface = Black1,
+    onSurface = Color.White,
+)
+
+@Composable
+fun AppTheme(
+    darkTheme: Boolean,
+    content: @Composable () -> Unit,
+) {
+    MaterialTheme(
+        colors = if (darkTheme) DarkThemeColors else LightThemeColors,
```

Comment:
You might consider the dynamic color theming as well. This [tutorial](https://developer.android.com/codelabs/jetpack-compose-theming?continue=https%3A%2F%2Fdeveloper.android.com%2Fcourses%2Fpathways%2Fjetpack-compose-for-android-developers-2%23codelab-https%3A%2F%2Fdeveloper.android.com%2Fcodelabs%2Fjetpack-compose-theming#4) should help you to implement it.